src/commands/txCommands/util/gitHelpers.js

```diff
@@ -0,0 +1,31 @@
+const chalk = require('chalk');
+const child_process = require('child_process');
+const Rx = require('rxjs');
+
+const child_process$ = Rx.Observable.bindNodeCallback(child_process.exec);
+
+/**
+ * Creates git commit of what has been staged
+ * @param  {String} resource resource slug to be used when we pull translations
+ * @param  {String} commitMessage what commit message to commit
+ * @return {Observable} Observable of tx process
+ */
+const commit$ = (resource, commitMessage) => {
```

Comment:
sort of a general comment, but this command might be simpler as a native Promise rather than observable, but that refactor will probably happen later anyway